data/i18n/bn/bn.toml

```
@@ -29,29 +53,32 @@ other = "Slack"
 other = "Stack Overflow"
 
 [community_twitter_name]
-other = "Twitter"
+other = "X (আগে টুইটার)"
+
+[community_x_name]
+other = "X (আগে টুইটার)"
 
 [community_youtube_name]
-other = "YouTube"
+other = "ইউটিউব"
 
 ## Avoid using conjunction_1.
 ## Must match the context in layouts/shortcodes/release-data.html
 ## Appears on https://kubernetes.io/releases/
 ## For example the "and" in "Complete 1.25 Schedule and Changelog"
 [conjunction_1]
-other = "and"
+other = "এবং"
 
 [cve_id]
 other = "CVE ID"
 
 [cve_issue_url]
-other = "CVE GitHub Issue URL"
+other = "CVE গিটহাব ইস্যু URL"
 
 [cve_summary]
-other = "Issue Summary"
+other = "ইস্যু সারাংশ"
 
 [cve_table]
-other = "Official Kubernetes CVE List"
+other = "অফিসিয়াল কুবারনেটিস CVE তালিকা"
 
 [cve_table_date_format]
 other = "02 Jan 2006 15:04:05 MST"
```

Can change this format, BTW. How about
```suggestion
other = "2006-01-02 15:04:05 MST"
```

---

it's depend on  en team , 

In Bangladesh we use 02 Jan 2006  or 02-02-2006  ,
Also most of the people here aware of that people around the world use  YYYY-MM-DD or MM-DD-YYYY

what's your suggestion

---

oh I [understand](https://github.com/kubernetes/website/pull/46271/files#r1595126057) ,,